docs/conf.py

```diff
@@ -76,6 +77,9 @@
     "doc_path": "docs",
 }
 
+jupyter_sphinx_require_url = ''
```

Comment:
this needs to be set to `null` so that jupyter-sphinx doesn't load its own version of requirejs